runner/runner.go

```diff
@@ -726,8 +725,12 @@ retry:
 		}
 	} else {
 		// Create a copy on the fly of the request body
-		bodyBytes, _ := req.BodyBytes()
-		req.Request.Body = ioutil.NopCloser(bytes.NewReader(bodyBytes))
+		var bodyBytes []byte
```

Comment:
I think it's possible to remove `bodyBytes` and use directly `scanopts.RequestBody`